pkgs/development/libraries/speechd/default.nix

```diff
@@ -1,38 +1,88 @@
-{ fetchurl, lib, stdenv, intltool, libtool, pkgconfig, glib, dotconf, libsndfile
-, libao, python3Packages
-, withEspeak ? false, espeak
+{ stdenv, lib, pkgconfig, fetchurl, wrapPython
+, intltool, itstool, libtool, texinfo, autoreconfHook
+, glib, dotconf, libsndfile
+, pyxdg
+, withLibao ? true, libao
+, withPulse ? false, libpulseaudio
+, withAlsa ? false, alsaLib
+, withOss ? false
+, withFlite ? true, flite
+, withFestival ? false # TODO: , festival-freebsoft-utils
+, withEspeak-ng ? true, espeak-ng, sonic, pcaudiolib
 , withPico ? true, svox
+, withIvona ? false # TODO: , libdumbtts
+, defaultModule ? null
+, defaultLang ? "en"
 }:
 
-stdenv.mkDerivation rec {
+with lib;
+let
+  selectedDefaultModule =
```

Comment:
Please add here a comment like “speechd hard-codes `espeak` even when built without a support for it.”

Reply:
Done